src/main/java/com/codeborne/selenide/SelenideElement.java

```diff
@@ -106,95 +114,114 @@
    * @return The innerText of this element
    * @see com.codeborne.selenide.commands.GetText
    */
-  @Override String getText();
+  @Nullable
+  @Override
+  String getText();
 
   /**
    * Short form of getText()
+   *
    * @see WebElement#getText()
    * @see com.codeborne.selenide.commands.GetText
    */
   @CheckReturnValue
+  @Nullable
   String text();
 
   /**
    * Get the text code of the element with children.
-   *
+   * <p>
    * Short form of getAttribute("textContent") or getAttribute("innerText") depending on browser.
+   *
    * @see com.codeborne.selenide.commands.GetInnerText
    */
   @CheckReturnValue
+  @Nullable
   String innerText();
 
   /**
    * Get the HTML code of the element with children.
-   *
+   * <p>
    * Short form of getAttribute("innerHTML")
+   *
    * @see com.codeborne.selenide.commands.GetInnerHtml
    */
   @CheckReturnValue
+  @Nullable
   String innerHtml();
 
   /**
    * Get the attribute of the element. Synonym for getAttribute(String).
+   *
    * @return null if attribute is missing
    * @see com.codeborne.selenide.commands.GetAttribute
    */
   @CheckReturnValue
+  @Nullable
   String attr(String attributeName);
 
   /**
    * Get the "name" attribute of the element
+   *
    * @return attribute "name" value or null if attribute is missing
    * @see com.codeborne.selenide.commands.GetName
    */
   @CheckReturnValue
+  @Nullable
   String name();
 
   /**
    * Get the "value" attribute of the element
    * Same as {@link #getValue()}
-   * @return attribute "value" value or null if attribute is missing
    *
+   * @return attribute "value" value or null if attribute is missing
    * @see com.codeborne.selenide.commands.Val
    */
   @CheckReturnValue
+  @Nullable
   String val();
 
   /**
    * Get the "value" attribute of the element
-   * @return attribute "value" value or null if attribute is missing
-   * @since 3.1
    *
+   * @return attribute "value" value or null if attribute is missing
    * @see com.codeborne.selenide.commands.GetValue
+   * @since 3.1
    */
   @CheckReturnValue
+  @Nullable
   String getValue();
 
   /**
    * Get the property value of the pseudo-element
+   *
    * @param pseudoElementName pseudo-element name of the element,
    *                          ":before", ":after", ":first-letter", ":first-line", ":selection"
-   * @param propertyName property name of the pseudo-element
+   * @param propertyName      property name of the pseudo-element
    * @return the property value or "" if the property is missing
    */
   @CheckReturnValue
+  @Nullable
```

Comment:
javadoc says `@return the property value or "" if the property is missing`
is it really nullable? if it is true please fix javadoc then